src/components/views/auth/signUp/SignUp.js

```diff
@@ -128,21 +140,25 @@ SignUp.propTypes = {
   onSubmit: PropTypes.func.isRequired,
   errors: PropTypes.shape({}),
   auth: PropTypes.shape({}),
+  social: PropTypes.shape({}),
   history: PropTypes.shape({}).isRequired,
   loading: PropTypes.bool.isRequired,
 };
 SignUp.defaultProps = {
   errors: {},
   auth: {},
+  social: {},
 };
 
 const mapStateToProps = state => ({
   auth: state.auth,
   errors: state.auth.errors,
   loading: state.auth.loading,
+  social: state.social,
 });
 const mapDispatchToProps = dispatch => ({
   onSubmit: (newUser, history) => dispatch(registerAccount(newUser, history)),
+  socialSignOn: token => dispatch(socialAuthPath(token)),
```

Comment:
This is the problem with variable names that are not descriptive(`socialAuthPath`), I couldn't tell what the dispatched action does just by the name